zoloto/utils.py

```diff
@@ -2,6 +2,14 @@
 from functools import wraps
 from typing import Any, Callable, TypeVar
 
+try:
+    import ujson
+
+    encode_as_json = ujson.dumps
+except ImportError:
+    encode_as_json = json.dumps  # type: ignore
```

Comment:
Worth noting that `json` is always imported, regardless of if it is used. Possible performance hit from this.

Reply:
I'm less worried about the import time performance. The main thing was the runtime performance, as `ujson` was previously imported at call time (however cached CPython makes this I don't know)